### PR TITLE
2755: Fixed index error in hidden content module

### PR DESCRIPTION
**Pull request by cableman**
#### Link to ticket

https://leantime.itkdev.dk/dashboard/home?tab=ticketdetails#/tickets/showTicket/2755

#### Description

Upgrades....

#### Screenshot of the result

N/A

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist, you have to add a comment explaining why this change
should be exempt from the list.

#### Additional comments or questions

N/A

### CODE REVIEW

File: config/sync/block.block.ai_launchpad_chatintegrationblock.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ settings:
   provider: chat
   provider_name: vllm
   models:
-    meta-llama/Meta-Llama-3-8B-Instruct: meta-llama/Meta-Llama-3-8B-Instruct
+    meta-llama/Llama-3¤1-8B-Instruct: meta-llama/Llama-3.1-8B-Instruct
```
Comment (jekuaitk) [inline config/sync/block.block.ai_launchpad_chatintegrationblock.yml:23]: `¤` looks funky

File: config/sync/redirect_after_login.settings.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 login_redirection:
   anonymous: null
-  authenticated: /
-  administrator: /
-  editor: /
+  authenticated: /node/26
+  administrator: /node/26
+  editor: /node/26
```
Comment (jekuaitk) [inline config/sync/redirect_after_login.settings.yml:5]: This may match the intended node in production, but it very well may not on staging/locally?

**Review by jekuaitk — APPROVED** [review]

File: config/sync/block.block.ai_launchpad_chatintegrationblock.yml (hunk 1 above)
Comment (cableman) [reply config/sync/block.block.ai_launchpad_chatintegrationblock.yml:23]: yes, but it works... the model name comes form vllm

File: config/sync/redirect_after_login.settings.yml (hunk 2 above)
Comment (cableman) [reply config/sync/redirect_after_login.settings.yml:5]: It works as prod is synced down

**Pull request merged**